### ISSUE TITLE
"Reviewed" Elements losing "Reviewed" Status 

### ISSUE THREAD

**Issue by mei215** [spawaskar-cora/cora-docs#645]
Elements that have been added, inventoried, and reviewed during the inventory process lose their "Reviewed" status sometimes when new data is added to some elements. I think* this happens when data is added to an element by the same person who inventoried or reviewed it initially, but I'm not sure. Once an element is "Reviewed," it  should stay reviewed (see attached inventory chart).

![1](https://user-images.githubusercontent.com/33871387/106498466-f8f1f800-6484-11eb-824e-f256a952e933.JPG)

PROD environment, Edge

**Comment by fedamann** [comment]
Thank you, Megan. Good investigation and presentation. If I understand
you're looking to have Reviews to change from the default of false to true
(when the review is first completed) and then never revert: true then
becomes immutable.

One question: do updates to a specimen record ever require an additional
peer review to validate the updated record?

**Comment by SachinPawaskarUNO** [comment]
@mei215 Agree, need to investigate why this is happening or what are the steps that are causing it to revert.

**Comment by mei215** [comment]
@fedamann, other projects (@clegarde, @Megyesimm, @veronicak7) may use the "Reviewed" button differently? But I have used it  to indicate that the peer review of that element in the initial inventory is complete, and thus, to track the progress of an overall project inventory. Later, when DNA sequence data is received and entered to elements to start forming individuals, that data is not initially peer reviewed in CoRA, but later in the CHR process. So I think once "Reviewed" an element should always stay "Reviewed." As is now, it just looks like peer reviewed elements have not been peer reviewed.
